### ISSUE TITLE
example error

### ISSUE THREAD

**Issue by alfhan** [buchin/kurs#2]
$response =  json_decode(file_get_contents('http://kurs.dropsugar.com/rates/bca.json'));
echo "Terakhir diupdate pada: " . date('j-m-Y H:i:s');
foreach ($response->kurs as $currency => $value) {
    echo 'Nilai jual ' . $currency . ' adalah: ' . $value->jual . '<br>';
    echo 'Nilai beli ' . $currency . ' adalah: ' . $value->beli . '<br><br>';
}

 Warning: file_get_contents(): php_network_getaddresses: getaddrinfo failed: No such host is known. in C:\xampp\htdocs\coba\kurs.php on line 2

**Comment by buchin** [comment]
Yeah that link no longer worked. I should put example in another url. Thank you for notifying me about this.
